docs/index.md

```diff
@@ -196,9 +196,25 @@ For more information, see the [settings documentation](settings.md).
 
 ### Running programmatically
 
-To run uvicorn directly from your application...
+There are many ways to run uvicorn directly from your application.
```

Comment:
"Many" feels a bit overwhelming — as we document here, there are only 3 ways (run / config-serve.run() / config-serve.serve()).

```suggestion
There are several ways to run uvicorn directly from your application.
```